package.json

```diff
@@ -2078,16 +2081,17 @@
     "dependencies": {
         "@enonic/fnv-plus": "^1.3.0",
         "@fluentui/react": "^7.160.1",
-        "@jupyter-widgets/base": "^4.0.0",
-        "@jupyter-widgets/controls": "^3.0.0",
-        "@jupyter-widgets/schema": "^0.4.0",
-        "@jupyterlab/coreutils": "^5.1.17",
-        "@jupyterlab/nbformat": "^3.1.17",
-        "@jupyterlab/services": "^6.1.17",
+        "@jupyter-widgets/base": "4.0.0",
```

Comment:
When regenerating the package-lock.json, I had to force these to a specific version (or it picks the latest non major one). 

Looks like jupyter changed their API again, so we can't just take the latest.